<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArtistInfoWidget</class>
 <widget class="QWidget" name="ArtistInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="handleWidth">
      <number>1</number>
     </property>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>1</number>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="HeaderLabel" name="label">
          <property name="text">
           <string>Top Hits</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="PlaylistView" name="topHits">
          <property name="headerHidden">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget1">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="HeaderLabel" name="label_2">
          <property name="text">
           <string>Related Artists</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ArtistView" name="relatedArtists">
          <property name="headerHidden">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="layoutWidget2">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="HeaderWidget" name="albumHeader" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="HeaderLabel" name="label_3">
            <property name="text">
             <string>Albums</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="ArtistView" name="albums"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlaylistView</class>
   <extends>QTreeView</extends>
   <header>playlist/PlaylistView.h</header>
  </customwidget>
  <customwidget>
   <class>HeaderLabel</class>
   <extends>QLabel</extends>
   <header location="global">widgets/HeaderLabel.h</header>
  </customwidget>
  <customwidget>
   <class>HeaderWidget</class>
   <extends>QWidget</extends>
   <header location="global">widgets/HeaderWidget.h</header>
  </customwidget>
  <customwidget>
   <class>ToggleButton</class>
   <extends>QPushButton</extends>
   <header location="global">widgets/ToggleButton.h</header>
  </customwidget>
  <customwidget>
   <class>ArtistView</class>
   <extends>QTreeView</extends>
   <header location="global">ArtistView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
